pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml

```diff
@@ -2,6 +2,12 @@ apiVersion: v1
 data:
   kube.conf: |-
     # Fluentd config file for OMS Docker - cluster components (kubeAPI)
+    #fluent forward plugin
```

Comment:
Please make sure this file matches the file [here] https://github.com/microsoft/Docker-Provider/blob/ci_feature_prod/installer/conf/kube.conf

Reply:
thank you. done in the latest commit